aviary/modules/processor.py

```diff
@@ -112,16 +112,32 @@ def __init__(self,
             self.refinery_max_iterations = args.refinery_max_iterations
             self.refinery_max_retries = args.refinery_max_retries
             self.skip_abundances = args.skip_abundances
+            self.skip_taxonomy = args.skip_taxonomy
+            self.skip_singlem = args.skip_singlem
+            if args.binning_only:
+                self.skip_abundances = True
+                self.skip_taxonomy = True
+                self.skip_singlem = True
+            self.binning_only = args.binning_only

```

Comment:
Wait, you do this. So why check for binning_only in the snakemake?

Reply:
Yeah, I was meant to clean up the snakefiles after testing worked but didn't get to it yesterday